packages/tools/client-debugger/client-debugger/src/FluidClientDebugger.ts

```diff
@@ -177,6 +184,31 @@ export class FluidClientDebugger
 			}
 			return false;
 		},
+		["CONNECT_CONTAINER"]: (untypedMessage) => {
+			const message = untypedMessage as ConnectContainerMessage;
+			if (message.data.containerId === this.containerId) {
+				this.container.connect();
+				return true;
+			}
+			return false;
+		},
+		["DISCONNECT_CONTAINER"]: (untypedMessage) => {
+			const message = untypedMessage as DisconnectContainerMessage;
+			if (message.data.containerId === this.containerId) {
+				this.container.disconnect(/* TODO: Specify debugger reason here once it is supported */);
+				return true;
+			}
+			return false;
+		},
+		["CLOSE_CONTAINER"]: (untypedMessage) => {
+			const message = untypedMessage as CloseContainerMessage;
+			if (message.data.containerId === this.containerId) {
+				this.container.close(/* TODO: Specify debugger reason here once it is supported */);
+				this.emit("debuggerClosed", this.containerId);
```

Comment:
I don't think we want this. We're closing the Container, but not the debugger. This event indicates that the debugger itself has been closed (not the container). 
```suggestion
```

Reply:
Yea..I through about it. Since the container is closed, there will be no history/log/dds update from it so our debugger will not get any updates. I might can also close the debugger as well. But yes, we can keep the debugger until the session ends.

Reply:
I see. I still think there is value in showing the container closed state to the user, so I think keeping the debugger alive is valuable to communicate that state. Otherwise the debugger entry would just disappear when the user closes the container, which may not be expected.

Reply:
Then in this case, I think we should also remove "The Container has been closed." check from Client debugger plane. It will clear out all pages once the container is closed.

Reply:
Yeah, that's probably a good idea. Especially now that `closed` and `disposed` are now distinct from one another. In the `closed` state, we probably want to keep displaying information. When the container gets to the `disposed` state, we may still want to show some disclaimer (i.e. keep the debugger alive), but I think at that point all other state can be considered stale / invalid, so we probably don't want to display it in that case.

Reply:
Make sense! Updated.